api/src/main/java/me/lokka30/treasury/api/economy/account/BankAccount.java

```diff
@@ -24,89 +24,137 @@
  * balance shared by multiple Bank Members, such as a 'town balance'
  * or 'faction balance'. The linked Player can be changed at any
  * time (Owning Player).
- *
  * @author lokka30, MrNemo64
  * @see Account
  * @since v1.0.0
  */
-@SuppressWarnings({"unused"})
+@SuppressWarnings({ "unused" })
```

Comment:
I'm unsure whether we should remove these warning suppressions from the project, and instead disable it for Treasury specifically in our IDE (disable inspection button).

Reply:
Remove all `@SuppressWarnings` is the go for a cleaner code.

Reply:
My suggestion here is outside of the scope of this PR, I've instead created an issue which I will work on once this PR is merged.